src/Stryker.Core/Stryker.Core.UnitTest/Compiling/CompilingProcessTests.cs

```diff
@@ -319,5 +327,100 @@ public int Subtract(int first, int second)
                 Assembly.Load(ms.ToArray()).GetName().Version.ToString().ShouldBe("0.0.0.0");
             }
         }
+
+        [Fact]
+        public void ShouldCompileAndRollbackErrorsForEventHandler()
```

Comment:
Very nice regression test!

Reply:
thanks. I wished I wrote this one sooner, it was sorely needed